package-lock.json

```diff
@@ -8751,267 +8751,6 @@
         "node": ">= 8"
       }
     },
-    "node_modules/@parcel/watcher": {
-      "version": "2.4.0",
-      "resolved": "https://registry.npmjs.org/@parcel/watcher/-/watcher-2.4.0.tgz",
-      "integrity": "sha512-XJLGVL0DEclX5pcWa2N9SX1jCGTDd8l972biNooLFtjneuGqodupPQh6XseXIBBeVIMaaJ7bTcs3qGvXwsp4vg==",
-      "hasInstallScript": true,
-      "dependencies": {
-        "detect-libc": "^1.0.3",
-        "is-glob": "^4.0.3",
-        "micromatch": "^4.0.5",
-        "node-addon-api": "^7.0.0"
-      },
-      "engines": {
-        "node": ">= 10.0.0"
-      },
-      "funding": {
-        "type": "opencollective",
-        "url": "https://opencollective.com/parcel"
-      },
-      "optionalDependencies": {
-        "@parcel/watcher-android-arm64": "2.4.0",
-        "@parcel/watcher-darwin-arm64": "2.4.0",
-        "@parcel/watcher-darwin-x64": "2.4.0",
-        "@parcel/watcher-freebsd-x64": "2.4.0",
-        "@parcel/watcher-linux-arm-glibc": "2.4.0",
-        "@parcel/watcher-linux-arm64-glibc": "2.4.0",
-        "@parcel/watcher-linux-arm64-musl": "2.4.0",
-        "@parcel/watcher-linux-x64-glibc": "2.4.0",
-        "@parcel/watcher-linux-x64-musl": "2.4.0",
-        "@parcel/watcher-win32-arm64": "2.4.0",
-        "@parcel/watcher-win32-ia32": "2.4.0",
-        "@parcel/watcher-win32-x64": "2.4.0"
-      }
-    },
-    "node_modules/@parcel/watcher-android-arm64": {
-      "version": "2.4.0",
```

Comment:
Can you run e2e on this PR? These platform builds are required in package-lock as far as I remember.